io-uring-test/src/tests/net.rs

```diff
@@ -1436,6 +1437,163 @@ pub fn test_udp_recvmsg_multishot<S: squeue::EntryMarker, C: cqueue::EntryMarker
 
     Ok(())
 }
+pub fn test_udp_recvmsg_multishot_trunc<S: squeue::EntryMarker, C: cqueue::EntryMarker>(
+    ring: &mut IoUring<S, C>,
+    test: &Test,
+) -> anyhow::Result<()> {
+    require!(
+        test;
+        test.probe.is_supported(opcode::RecvMsgMulti::CODE);
+        test.probe.is_supported(opcode::ProvideBuffers::CODE);
+        test.probe.is_supported(opcode::SendMsg::CODE);
+    );
+
+    // Check kernel version
+    {
+        let mut uname: libc::utsname = unsafe { std::mem::zeroed() };
```

Comment:
This isn't the best solution in my opinion, but I can live with it.

Can you wrap it into a function like this?

```rust
fn check_kernel_version(version: &str) -> bool;
```

then put it into `require!` macro.

Reply:
> This isn't the best solution in my opinion

I agree, it's ugly lol.

---

Ok actually the new require macro isn't half bad.